### DISCUSSION TITLE
Authenticator - Naming convention

### DISCUSSION THREAD

**Discussion by santhoshit8** [webauthn4j/webauthn4j#716] [General]
Hi @ynojima ,

Webauthn L3 call public key related data as "[Credential Record](https://w3c.github.io/webauthn/#ref-for-credential-record)"

Webauthn4j call it  as "Authenticator".

Most other library also call it is "Credential"

do you have any idea to name change of webbauthn4j's "Authenticator".

**Comment by ynojima** [comment]
"Credential" is more preferable, but it is hard to rename from now :(
